package.json

```diff
@@ -24,7 +24,7 @@
   "author": "",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.5.0"
+    "node": ">=12.13.0"
```

Comment:
- If we want to support Node.js 12 in general, it might just be better to declare `>=12`.
- If 12.13.0 was chosen because it was the first LTS version of Node.js 12, the ideas conveyed in the documentation should be shifted to reflect the slight difference therein ("Node.js 12" to "LTS versions of Node.js 12" or something of that nature).

Reply:
Do you have any recommendations on which one would be better? I'm personally fine with either, as long as it's a version supporting all the features we use in the client. I'm not very familiar with npm packaging in general, so I'm happy to defer to your judgment here.

Reply:
If I understand your [subsequent comment](https://github.com/m-lab/ndt7-js/pull/46#issuecomment-842773115) correctly, `>= 12` and the current phrasing in the README seems ok.